docs/iris/src/whatsnew/latest.rst

```diff
@@ -13,193 +13,255 @@ This document explains the changes made to Iris for this release
   The new routines perform the same operation, but in a style more like other
-  ``Iris`` functions such as :meth:`~iris.load_cube` and :meth:`~iris.load_cubes`.
+  ``Iris`` functions such as :meth:`~iris.load_cube` and:meth:`~iris.load_cubes`.
   Unlike ``strict`` extraction, the type of return value is now completely
   consistent : :meth:`~iris.cube.CubeList.extract_cube` always returns a
   :class:`~iris.cube.Cube`, and :meth:`~iris.cube.CubeList.extract_cubes`
   always returns an :class:`iris.cube.CubeList` of a length equal to the
-  number of constraints.
+  number of constraints. (:pull:`3715`)
 
-* The former function ``iris.analysis.coord_comparison`` has been removed.
+* `Patrick Peglar`_ removed the former function
+  ``iris.analysis.coord_comparison``. (:pull:`3562`)
 
-* The :func:`iris.experimental.equalise_cubes.equalise_attributes` function
-  has been moved from the :mod:`iris.experimental` module into the
-  :mod:`iris.util` module.  Please use the :func:`iris.util.equalise_attributes`
-  function instead.
+* `Bill Little`_ moved the
+  :func:`iris.experimental.equalise_cubes.equalise_attributes` function from
+  the :mod:`iris.experimental` module into the :mod:`iris.util` module.  Please
+  use the :func:`iris.util.equalise_attributes` function instead.
+  (:pull:`3527`)
 
-* The :mod:`iris.experimental.concatenate` module has now been removed. In
+* `Bill Little`_ removed the :mod:`iris.experimental.concatenate` module. In
   ``v1.6.0`` the experimental ``concatenate`` functionality was moved to the
   :meth:`iris.cube.CubeList.concatenate` method.  Since then, calling the
   :func:`iris.experimental.concatenate.concatenate` function raised an
-  exception.
+  exception. (:pull:`3523`)
 
-* When loading data from NetCDF-CF files, where a variable has no ``units``
-  property, the corresponding Iris object will have ``units='unknown'``.
-  Prior to Iris ``3.0.0``, these cases defaulted to ``units='1'``.
-  See :pull:`3795`.
+* `stephenworsley`_ changed Iris objects loaded from NetCDF-CF files to have
+  ``units='unknown'`` where the corresponding NetCDF variable has no ``units``
+  property. Previously these cases defaulted to ``units='1'``. (:pull:`3795`)
 
 
-Internal
-========
+Deprecations
+============
 
-* Changed the numerical values in tests involving the Robinson projection due
-  to improvements made in `Proj <https://github.com/OSGeo/PROJ>`_ (see
-  `proj#1292 <https://github.com/OSGeo/PROJ/pull/1292>`_ and
-  `proj#2151 <https://github.com/OSGeo/PROJ/pull/2151>`_).
+* `stephenworsley`_ removed the deprecated :class:`iris.Future` flags
+  ``cell_date_time_objects``, ``netcdf_promote``, ``netcdf_no_unlimited`` and
+  ``clip_latitudes``. (:pull:`3459`)
 
-* Change tests to account for more detailed descriptions of projections in
-  `GDAL <https://github.com/OSGeo/gdal>`_ 
-  (`see GDAL#1185 <https://github.com/OSGeo/gdal/pull/1185>`_).
+* `stephenworsley`_ changed :attr:`iris.fileformats.pp.PPField.lbproc` is be an
```

Comment:
"is be an" should be "to be an"

Reply:
d1038c403d23de9e461e3fe5e3a94cb7def73778